pkg/telemetryquerylanguage/functions/tqlcommon/README.md

```diff
@@ -27,6 +28,30 @@ Examples:
 
 - `IsMatch("string", ".*ring")`
 
+## Int
+
+`Int(value)`
+
+The `Int` factory function converts the `value` to int type.
+
+The returned type is int64.
+
+The input `value` types:
```

Comment:
Technically int64 is a supported type. Might want to state somewhere that passing int64 just returns the input with no changes.